openhtf/plugs/__init__.py

```diff
@@ -116,6 +116,15 @@ def __init__(self, my_config_key)
 
 _LOG = logging.getLogger(__name__)
 
+PlugDescriptor = collections.namedtuple('PlugDescriptor', ['mro'])  # pylint: disable=invalid-name
+
+# Placeholder for a specific plug to be provided before test execution.
+#
+# Use the with_plugs() method to provide the plug before test execution. The
+# with_plugs() method checks to make sure the substitute plug is a subclass of
+# the PlugPlaceholder's base_class.
+PlugPlaceholder = collections.namedtuple('PlugPlaceholder', ['base_class'])  # pylint: disable-invalid-name
```

Comment:
why did you need to move the docstring into a comment?

Reply:
Only since it seemed weird to have two different styles in use:
```python
PlugDescriptor = collections.namedtuple(...)
```
and
```python
class PlugPlaceholder(collections.namedtuple(...)):
   ...
```
And since the former seems to be the dominant style, except when something else is added to the class.

The class-style was nice for the long docstring so I could see this going either way, but I opted for what I thought was more consistent. Happy to revert this if you have an opinion about it.